js/app.js

```diff
@@ -1 +1,80 @@
-// Your code here
+// Model
+
+$(document).ready(function() {
+
+var model = {
+      loggedIn: false,
+      userId: '',
+      username: '',
+      category: [],
+      articles: []
+  };
+
+// View
+
+function renderUser() {
+  var templateSource = $('#user-template').html();
+  var template = Handlebars.compile(templateSource);
+  var templateRender = template(model);
+  $('#loginContainer').html(templateRender);
+};
+
+renderUser();
+
+// Controller
+// event listners and ajax
+
+$('#loginSubmit').on('click', addUser);
+
+function addUser() {
+  var user = $('#username').val();
+  $.ajax {
+    url: '/login',
+    type: POST,
+    data: { username: username },
+  }
+}
+
+function renderArticles() {
+  var templateSource = $('#article-template').html();
+  var template = Handlebars.compile(templateSource);
+  var templateRender = template(model);
+  $('#main').html(templateRender);
+};
+
+renderArticles();
```

Comment:
just from an organizational perspective, I like to group all of the things that run right away together. It helps me see how the application gets set up. Each of these render function calls and the attachment of event listeners (like on line 27) happen as soon as the document is ready, so it might be nice to keep them together.